storage/testonly/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testonly contains fakes for use in tests that interact with storage.
+package testonly
```

Comment:
Does this really need to be in its own file right now, when there's only one file with it?  I'd say put this doc comment in the roots.go one, and we can pull it out into a separate file if/when we end up adding more files in here.